server/api/itemRouter.js

```diff
@@ -0,0 +1,65 @@
+const Item = require('../db/models/item')
+const User = require('../db/models/user')
+const Category = require('../db/models/category')
+const express = require('express')
+const mongoose = require('mongoose');
+
+const router = express.Router()
+
+router.post('/addItem', (req, res) => {
+    const body = req.body;
+
+    if(!body) {
+        next(err);
+    }
+
+    const item = new Item(body);
+
+    if(!item) {
+        next(err);
+    }
+
+    item.save().then(() => {
+        res.status(201).json({
+            success: true,
+            message: 'Item added'
+        })
+    });
+})
+
+router.get('/getItems', async (req, res) => {
+    await User.find({_id:req.body.userId }, (err, item) => {
```

Comment:
don't think await is needed here?

Reply:
oh and findOne() for consistency?

Reply:
ah I seeeee! you right cuz all items are returned at once